### DISCUSSION TITLE
Testimonials

### DISCUSSION THREAD

**Discussion by westonwalker** [spark-dotnet/framework#12] [General]
I'd like to add a testimonials section to the doc sites. If you've used the framework and have some nice things to say about, please leave a quick testimonial here.

Testimonial Template:
Name:
Job title + Company or Project:
Testimonial:

Ex:
Name: Weston Walker
Job title + Company or Project: Software Developer. Creator of Blazor Spark
Testimonial: Spark takes the pain out of building modern, scalable web apps.

**Comment by trifleneurotic** [comment]
Name: Gabe Stone
Job title + Company or Project: Freelance Developer, custom boutique property management apps
Testimonial: Spot-on, useful features out-of-the-box, MVP in much less time!
